wiz-sensor/templates/daemonset.yaml

```diff
@@ -360,7 +360,7 @@ spec:
           value: {{ .Values.tenantSuffix }}
         {{- end }}
         - name: SENSOR_CLUSTER_NAME
-          value: {{ .Values.sensorClusterName }}
```

Comment:
you can't change the sensor helm chart from here. it has to be via gerrit from our git. but changes look ok...

Reply:
should we take them via our git?

Reply:
Yes please :-)